### DISCUSSION TITLE
Wifi interference. How do I switch Zigbee channel?

### DISCUSSION THREAD

**Discussion by dinki** [Koenkk/zigbee2mqtt#17992] [General]
Hey guys. I just installed some mesh wifi routers which, surprising to me, does not allow me to change the wifi channel and I THINK I'm getting interference on my zigbee radio. My zigbee set up was rock solid for a long time before making the wifi change. Now I have devices dropping off daily whereas before I had zero problems

Wifi analyzer app is showing my 2.4GZ on channels 1-7 . I tried changing z2mqtt to use channel 25 with the dread anticipation of having to readded my devices. I tried making the change but my Docker container failed to start. Looking at the logs it was complaining about the switch from 11 to 25 and the need to update/repair the devices.

https://pastebin.com/Lnp2yb4G

Can someone tell me the proper way of changing the channel of z2mqtt to 25? After making this change can I expect the repaired devices to show up the same way they were before? I've got a ton of things referencing these devices as named entities in Home Assistant and it would be great to not have to go back and touch every one of these.

**Comment by lux73** [comment]
if you change Channel on z2m you have to re-pairing all Zigbee Devices

`Zigbee2MQTT:error 2023-06-10 13:57:59: If you wish to re-commission your network, please remove coordinator backup at /app/data/coordinator_backup.json.`

after doing this your z2m intance will start normally with your preferred Channel

**Comment by dinki** [comment]
@lux73  Thank you for the response.  I did manage to get this done yesterday.  Steps included, changing channel in z2m interface, stopping docker container, removing coordinator_backup.json file, restarting container.

The process of repairing was mostly painless as z2m remembered my device names and things were functioning as expected.

Posting the above to perhaps someone else that may stumble on this in the future.  Thanks for the help.
